metaflow/current.py

```diff
@@ -10,6 +10,7 @@ def __init__(self):
         self._namespace = None
         self._username = None
         self._is_running = False
+        self._parameter_names = None
```

Comment:
Should these be just parameters? Maybe the structure could be a dictionary of `names` and `values`?